audit-worker-versions/main.go

```diff
@@ -283,6 +283,12 @@ func createTasks(queue *tcqueue.Queue, taskGroupID string) {
 
 		fmt.Println(string(respJSON))
 	}
+	fmt.Println("Tasks created, sealing task group...")
+	tg, err := queue.SealTaskGroup(taskGroupID)
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.Printf("Task group sealed: %v", tg)
```

Comment:
```suggestion
	log.Printf("Task group sealed: %v", tg.TaskGroupID)
```

Reply:
I don't remember how `%v` renders for a struct, but with this suggestion it would just show the task group ID. But maybe you wanted to show the full struct. Another option is using `%#v` which would render it as a go type.

Reply:
Thanks, updated!